docs/application/web/api/5.5/device_api/tv/tizen/tvaudiocontrol.html

```diff
@@ -0,0 +1,847 @@
+<p><span class="version">Since: </span>
+ 2.3
+          </p>
+<div class="description">
+          <p>
+There will be a <em>tizen.tvaudiocontrol </em>object that allows the access to the
+Audio Control API.
+          </p>
+         </div>
+</div>
+<div class="interface" id="AudioControlManager">
+<a class="backward-compatibility-anchor" name="::TVAudioControl::AudioControlManager"></a><h3>2.2. AudioControlManager</h3>
+<div class="brief">
+ This interface provides access to the API funtionalities through the <em>tizen.tvaudiocontrol</em> interface.
+          </div>
+<pre class="webidl prettyprint">  [NoInterfaceObject] interface AudioControlManager {
+    void setMute(boolean mute) raises(<a href="tizen.html#WebAPIException">WebAPIException</a>);
+    boolean isMute() raises(<a href="tizen.html#WebAPIException">WebAPIException</a>);
+    void setVolume(unsigned short volume) raises(<a href="tizen.html#WebAPIException">WebAPIException</a>);
+    void setVolumeUp() raises(<a href="tizen.html#WebAPIException">WebAPIException</a>);
+    void setVolumeDown() raises(<a href="tizen.html#WebAPIException">WebAPIException</a>);
+    unsigned short getVolume() raises(<a href="tizen.html#WebAPIException">WebAPIException</a>);
+    void setVolumeChangeListener(<a href="#VolumeChangeCallback">VolumeChangeCallback</a> callback) raises(<a href="tizen.html#WebAPIException">WebAPIException</a>);
+    void unsetVolumeChangeListener() raises(<a href="tizen.html#WebAPIException">WebAPIException</a>);
+    <a href="#AudioOutputMode">AudioOutputMode</a> getOutputMode() raises(<a href="tizen.html#WebAPIException">WebAPIException</a>);
+    void playSound(<a href="#AudioBeepType">AudioBeepType</a> type) raises(<a href="tizen.html#WebAPIException">WebAPIException</a>);
+  };</pre>
+<p><span class="version">Since: </span>
+ 2.3
+          </p>
+<div class="methods">
+<h4>Methods</h4>
+<dl>
+<dt class="method" id="AudioControlManager::setMute">
+<a class="backward-compatibility-anchor" name="::TVAudioControl::AudioControlManager::setMute"></a><code><b><span class="methodName">setMute</span></b></code>
+</dt>
+<dd>
+<div class="brief">
+ Turns on or off the silent mode
+            </div>
+<div class="synopsis"><pre class="signature prettyprint">void setMute(boolean mute);</pre></div>
+<p><span class="version">Since: </span>
+ 2.3
+            </p>
+<div class="description">
+            <p>
+Note that turning on mute mode does not change volume level but
+it simply disables any sound. Turning off the mute will enable sound
+with the volume level. If setVolumeUp or setVolumeDown functions
+are used, then mute is disabled.
+            </p>
+           </div>
+<p><span class="privilegelevel">Privilege level: </span>
+ public
+            </p>
+<p><span class="privilege">Privilege: </span>
+ http://tizen.org/privilege/tv.audio
+            </p>
+<div class="parameters">
```

Comment:
It looks like #529 is missing in some files.
Is it OK not to add "deprecated since 5.0" information?